src/http-statements.md

```diff
@@ -24,10 +27,11 @@ timestamp | Mongo Date | When this statement's activity occured _(autogenerated)
 organisation | Mongo ID | The organisation this statement belongs in _(autogenerated)_
 lrs_id | Mongo ID | The store this statement belongs in _(autogenerated)_
 client | Mongo ID | The client that inserted this statement _(autogenerated)_
-person | [Person](#person) | Information about the person assoiated to the actor in this statement
+person | [Person](#person) | Information about the person associated to the actor in this statement
 voided | Boolean |  Has the statement been voided
 hash | String | A unique hash of the statement used for conflict checks on insert _(autogenerated)_
 statement | [Statement](#statement) | The statement object as constructed and validated by the xAPI service
+metadata | [Metadata](../http-metadata#schema) | The metadata related to the statement
```

Comment:
Nice, thanks for adding this and fixing the typo above. We haven't actually added a section with "Schema" as a header on that metadata page though, so we might want to add that.

Reply:
Oops, when I added this there was a schema on that page, but I took it out. I'll put it back in 👍